GSA/made-in-america-data
# Merge into Main

This PR covers: 
- csv download
- CONTRIBUTING.MD
- README.MD

To test:
 check and verify csv is formatted correctly. Should be in table format. The file name is waivers.csv
